core/mgmt/v1beta/mgmt_public_service.proto

```diff
@@ -324,6 +324,16 @@ service MgmtPublicService {
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {tags: "Metric"};
   }
 
+  // List model trigger time charts
+  //
+  // Returns a timeline of model trigger counts for a given requester. The
+  // response will contain one set of records (datapoints), representing the
+  // amount of triggers in a time bucket.
+  rpc ListModelTriggerChartRecords(ListModelTriggerChartRecordsRequest) returns (ListModelTriggerChartRecordsResponse) {
+    option (google.api.http) = {get: "/v1beta/metrics/vdp/model/model-runs:query-charts"};
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {tags: "Metric"};
```

Comment:
The tag has changed to `📊  Metrics`. Please rebase your branch and check the newly added contribution guidelines.

Reply:
rebased and updated tag